apps/src/code-studio/components/AdvancedShareOptions.jsx

```diff
@@ -44,8 +44,8 @@ const style = {
   },
 };
 
-const AdvancedShareOptions = Radium(React.createClass({
-  propTypes: {
+const AdvancedShareOptions = Radium(class extends React.Component {
```

Comment:
Out of scope for this PR, but have we considered adding decorator support and using `@Radium` to add Radium to our components?

Reply:
That would be awesome.  I loved using annotations in AS3.